src/app/expansion-panel/expansion-panel.component.html

```diff
@@ -9,6 +9,6 @@
     {{ expanded ? 'expand_less' : 'expand_more' }}
   </button>
 </div>
-<div [ngClass]="{ 'tce-hidden': !expanded }" [class]="config.contentContainerStyles" role="region">
+<div [ngClass]="{ 'tce-hidden': !expanded }" [class]="config.contentContainerStyles">
```

Comment:
I'm not sure why there was a region role here but I get a warning from SonarLint saying 'Use <section> instead of the region role to ensure accessibility across all devices.' so perhaps we shouldn't be using `role="region"` here regardless of whether a role is required. I would guess it would be down to the content that is passed in to have the correct roles so a `div` is ok here but I'm not sure.